_posts/2015-12-10-RGSoC.markdown

```diff
@@ -0,0 +1,65 @@
+
+With search finished there were still a couple of weeks left of the project. I wanted to learn more about web development and collaborated with [Orta](https://twitter.com/orta) on a new project called [CocoaPods Stories](https://github.com/CocoaPods/stories.cocoapods.org). The idea was to give Pod users the ability to write stories about what Pods they are using and how. We didn’t make it longer than the planning and research stage and we are not sure wether it will still be useful. Please ping us if you want us to continue.
+
+After working on CocoaPods-label with Boris and Emma, CocoaPods issue #3398 'Extract UI module into its own gem', labeled d1:easy, seemed particularly appealing. Samuel Giddins set up the project scaffolding, and we outlined the first steps to work on #3398 by breaking it down into smaller steps.
+To extract the User Interface module, CLAide into its own gem, so that it works in isolation of CocoaPods, we had to get rid of all CocoaPods specific language, dependencies and documentation. CocoaPods syntax was challenging to spot out initially. So, pairing with Samuel and Kyle Fuller at GitHub was crucial. After cleaning up CLAide, and freeing it of CP dependencies, writing tests for all of the left over User Interface code was the next step.
+
+For testing we used Rubocop, CodeCov, Code Climate and Travis CI diligently.
+After writing tests,and or changing tests, we had to debug a lot. Debugging felt like a never ending story, but it is an enjoyable one. We are still working on Cork, and I am looking forward to wrapping it up before Summer 2016 rolls around. Also, if you're curious about the name Cork, we  found it fitting to re-name the the CocoaPod free User Interface [Cork](https://github.com/CocoaPods/Cork), since Cork like CocoaPods, is also a commonly harvested part of the tree.
```

Comment:
> Also, if you're curious about the name Cork, we  found it fitting to re-name the the CocoaPod free User Interface [Cork](https://github.com/CocoaPods/Cork), since Cork like CocoaPods, is also a commonly harvested part of the tree.

-> Also, if you're curious about the name, we found it fitting to name the CocoaPods-free User Interface library [Cork](https://github.com/CocoaPods/Cork), since Cork like CocoaPods, is also a commonly harvested part of the tree.